packages/twenty-front/src/modules/object-record/utils/isObjectMetadataItemSearchable.ts

```diff
@@ -0,0 +1,17 @@
+import { ObjectMetadataItem } from '@/object-metadata/types/ObjectMetadataItem';
+
+const SEARCHABLE_STANDARD_OBJECTS_NAMES_PLURAL = [
+  'companies',
+  'people',
+  'opportunities',
+];
+export const isObjectMetadataItemSearchable = (
```

Comment:
I think this name is confusing because technically 'notes' are now searchable as well. If you want this util to be specific to multi objects search maybe we can find a better name? If it could work for notes then let's add it!